pyproject.toml

```diff
@@ -1,26 +1,27 @@
 [tool.poetry]
 name = "pelican"
-version = "0.6.5"
+version = "0.6.4"
 description = ""
 authors = ["CTDS UChicago <cdis@uchicago.edu>"]
 
 [tool.poetry.dependencies]
-python = ">=3.9,<3.10.0"
-pypfb = ">=0.5.16"
-fastavro = "^1.3.1"
+python = ">=3.9,<3.10"
+pypfb = ">=0.5.21"
+fastavro = "<=1.4.1"
 requests = "^2.25.1"
-psycopg2-binary = "^2.8.6"
+psycopg2-binary = ">=2.8.4"
 pyspark = "^3.0.1"
 psutil = "^5.8.0"
-boto3 = "~1.28.33"
+boto3 = "^1.26.0"
 gen3 = "^4.2.0"
 psqlgraph = "3.0.1"
-gen3datamodel = "<3.0.2"
-dictionaryutils = "^3.4.6"
-SQLAlchemy = "^1.4.9"
+gen3datamodel = ">=3.2.1"
+datamodelutils = ">=1.1.0"
+dictionaryutils = "^3.4.1"
+SQLAlchemy = ">=1.3,<2.0"
 
 [tool.poetry.dev-dependencies]
 
 [build-system]
-requires = ["poetry<1.2"]
+requires = ["poetry==1.8.4"]
```

Comment:
I think we try to stay away from using "==" but for poetry it's most likely fine.